src/main/java/use_case/TaskGetter.java

```diff
@@ -43,13 +45,21 @@
         return task_data;
     }
 
-    public Task getTaskByName(String name, AccessTodoData accessTodoData) {
-        List<Task> allTasks = accessTodoData.getTodoList().getTasks();
-        for (Task task : allTasks) {
-            if (task.getTaskName().equals(name)) {
-                return task;
+    public TaskInfo getTaskByName(String name) {
```

Comment:
Ideally we want to get rid of `get` type methods here, but we might present tasks by their id. The UI should store the relationship between visual objects and the ids of them so that it can use the id when making queries to the app.